docs/basics/101-4-summary.rst

```diff
@@ -0,0 +1,44 @@
+Starting from scratch: Summary
+------------------------------
+
+In the last few pages, we have discovered the basics of starting a DataLad dataset from scratch,
```

Comment:
s/pages/section -- makes it less dependent on the actual format that people consumed

Reply:
true, done.